packages/map/src/vaadin-lit-map.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See https://vaadin.com/commercial-license-and-service-terms for the full
+ * license.
+ */
+import { html, LitElement } from 'lit';
```

Comment:
Let's add `lit` as a direct dependency to `package.json` as we do for other components.

Reply:
Done